### PR TITLE
Adding missing message activity flags

### PR DESCRIPTION
**Pull request by Awoocado**
discord/discord-api-docs#3877

### CODE REVIEW

**Review by bsian03 — CHANGES_REQUESTED** [review]
They aren't bitshifted as according to docs

**Comment by Awoocado** [comment]
wait, do MessageActivityTypes in Constants need `<<` ?

File: index.d.ts [hunk 1]
```diff
@@ -1266,10 +1266,15 @@ declare namespace Eris {
       directMessageTyping: 16384;
     };
     MessageActivityTypes: {
-      JOIN: 1;
-      SPECTATE: 2;
-      LISTEN: 3;
-      JOIN_REQUEST: 5;
+      INSTANCE: 0;
```
Comment (bsian03) [inline index.d.ts]:
```suggestion
      INSTANCE: 1;
```

**Review by bsian03 — CHANGES_REQUESTED** [review]
Yes. They aren't types, they're flags (please don't rename for now, I'll have a look later at the implications of changing them)

**Comment by bsian03** [comment]
Feel free to change the names and I'll merge this in soon as

**Review by bsian03 — APPROVED** [review]

**Pull request merged**